File: 03-frontend-booking-system/booking-system-web/src/app/features/events/dtos/event-response-dto.ts
import { EventTypeModel } from "./event-type-model";

export class EventResponseDto {
    id!:number;
    eventName!:string;
    eventDescription?:string;
    eventLocationAddress!:string;
    includePosition!:boolean;
    latitude!:number | null;
    longitude!:number | null;
    maxBookingsPerIdentity!: number | null;
    eventType!: EventTypeModel;
    createdAt!: string;
    updatedAt!: string;
}

File: 02-backend-booking-system/src/main/java/com/littlek4za/booking_system/features/event/dto/EventResponseDto.java
package com.littlek4za.booking_system.features.event.dto;

import java.time.Instant;

public record EventResponseDto(
    Long id,
    String eventName,
    String eventDescription,
    String eventLocationAddress,
    Boolean includePosition,
    Double latitude,
    Double longitude,
    Integer maxBookingsPerIdentity,
    String eventType,
    Instant createdAt,
    Instant updateAt
) {}
